code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -51,6 +52,8 @@
 	potion_action = new
 	stored_slimes = list()
 	listener = AddComponent(/datum/component/redirect, list(COMSIG_ATOM_CONTENTS_DEL = CALLBACK(src, .proc/on_contents_del)))
+	connected_recycler = GLOB.monkey_recycler_default
```

Comment:
This makes it so ANY xenobio cam syncs to this recycler (including say ones in lavaland or space).

Consider a check similar to the recycler here.

Sorry for being nitpicky I wanna merge this but don't want to get bullied if something is wrong l0l

Reply:
That's a fair point, I'll make the glob a list, and iterate through and check that they are in the same area as the console before adding them. This will still allow manual connections, but roundstart initialization should never connect the wrong thing.

Reply:
Actually, looking back I made it so that only a monkey recycler on the station z-level is added to the glob. I'll still make the change though.

Reply:
correct, i'm just referring to the cams.

Reply:
get_area returns the area instance so you don't have to worry about name.

Area names might be duplicated, I haven't checked blueprints/make a new area to see if they allow for that.

Reply:
Right, yeah that makes sense. I am still thinking a bit backwards about the DM code in some cases, but this is indeed comparing the area instance which would make it unique.

This should be good to go then, I tested and it works for on-station consoles/recycler. There are currently not any xenobio setups that initialize on mapload anywhere else, so this is harder to test. Still, if it works on station it should work everywhere else too.